### ISSUE TITLE
Layout/IndentationWidth false positive after chain methods

### ISSUE THREAD

**Issue by seandilda** [rubocop/rubocop#14834]
1.84.1 seems to have introduced a false positive in `Layout/IndentationWidth` after chain methods

--------

## Steps to reproduce the problem

```ruby
# frozen_string_literal: true

y = []
LongClassName.method1
             .method2
             .each do |x|
  y << x
end
```

## Expected behavior

No offenses found

## Actual behavior

```
$ rubocop --debug test.rb
For /app: configuration from /app/.rubocop.yml
Default configuration from /home/docker-user/bundle/ruby/3.3.0/gems/rubocop-1.84.1/config/default.yml
Plugin configuration from /home/docker-user/bundle/ruby/3.3.0/gems/rubocop-rails-2.34.3/config/default.yml
Use parallel by default.
Skipping parallel inspection: only a single file needs inspection
Inspecting 1 file
Scanning /app/test.rb
Loading cache from /home/docker-user/.cache/rubocop_cache/f949e70474bdfbf9be0eb032b218ce426938c3fb/8c3e361d53661224764d286241e06b0a70c26148/5ed5e66f9a22606836a2a0b6255261fa5f9799f0
C

Offenses:

test.rb:7:3: C: [Correctable] Layout/IndentationWidth: Use 2 (not -11) spaces for indentation.
  y << x ...
  ^^^^^^

1 file inspected, 1 offense detected, 1 offense autocorrectable
Finished in 0.2037543149999692 seconds
```

## RuboCop version

```
rubocop -V
1.84.1 (using Parser 3.3.10.1, Prism 1.9.0, rubocop-ast 1.49.0, analyzing as Ruby 3.3, running on ruby 3.3.10) [x86_64-linux]
  - rubocop-rails 2.34.3
```

**Comment by lavaturtle** [comment]
This reminds me of #14747, did that fix get un-done somehow?

**Comment by bendilley** [comment]
I agree, this is just like #14747 and throws up loads of false positives (again) for my code.
